tox.ini

```diff
@@ -5,7 +5,10 @@ envlist = py34,py35,py36,pep8
 usedevelop = True
 install_command = pip install {opts} {packages}
 deps = -r{toxinidir}/test-requirements.txt
-commands = nosetests
+commands =
+    nosetests \
+    --with-timer \
+    --with-coverage --cover-erase --cover-package=kojak
```

Comment:
I think it's better to make this in another patch.

Reply:
see #20